.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no-install lint-staged
```

Comment:
Please add a comment, specifying what this line does

Reply:
Comments added

related commit - b57fef5f442d85fd03e9ab36cc91b055d10b1ae1